profiles/SampleMoonMage-setup.yaml

```diff
@@ -0,0 +1,136 @@
+---
+# https://elanthipedia.play.net/mediawiki/index.php/Moon_mage#Circle_Requirements
+# 4 Magics: Augmentation, Attunement, Arcana, Targeted Magic/Debilitation (Does not include Lunar Magic or Astrology)
+# 4 Survivals: Perception, Athletics, Evasion, Skinning (Does not include Thievery)
+# 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
+# 2 Additional: Astrology, Scholarship
+
+# It is recommended to get a target or debilitation spell for your second circle. (e.g. Dinazen Olkar or Calm)
```

Comment:
You might consider adding the circle 2 spell in the yaml, but commented out. You can then have a note here that says to remove the #s after learning it at circle 2.